resources/ext/ezbake.conf

```diff
@@ -21,8 +21,9 @@ ezbake: {
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master rundir  /var/run/puppetlabs/puppetserver",
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master pidfile /var/run/puppetlabs/puppetserver/puppetserver.pid",
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master codedir /etc/puppetlabs/code",
-                 "usermod --home /opt/puppetlabs/server/data/puppetserver puppet",
-                 "install --directory --owner=puppet --group=puppet --mode=775 /opt/puppetlabs/server/data"
+                 "install --directory /etc/puppetlabs/puppet/ssl",
+                 "chown -R puppet:puppet /etc/puppetlabs/puppet/ssl",
+                 "chmod -R 770 /etc/puppetlabs/puppet/ssl",
```

Comment:
I'm concerned about this line.  What happens if someone has already installed `puppet-agent`, then some time later decides to install `puppetserver`?  Will the permissions of all of the existing files be changed such that we might end up with executable PEM files and such?